code_examples/sdk_examples/src/core_features/messaging/01_generate_message.ts

```diff
@@ -0,0 +1,39 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+
+export async function generateMessage(
+  senderUri: Kilt.DidUri,
+  receiverUri: Kilt.DidUri,
+  cTypeHash: Kilt.CTypeHash
+) {
+  // Creating a challenge to submit to the receiver
+  const challenge = Kilt.Utils.UUID.generate()
+
+  // Sender uri is checked if it is a valid uri
+  Kilt.Did.validateUri(senderUri)
+  // Receiver uri is checked if it is a valid uri
+  Kilt.Did.validateUri(receiverUri)
+
+  // The content of the request credentials
+  // It includes a ctype that is being requested, this can be for attestation or verification
+  // The sender is the trusted attester in the scenario
+  const requestCredentialContent = {
+    cTypeHash: cTypeHash,
+    trustedAttesters: [senderUri]
+  }
+
+  const messageBody: Kilt.IRequestCredential = {
+    type: 'request-credential',
+    content: { cTypes: [requestCredentialContent], challenge: challenge }
```

Comment:
```suggestion
    content: { cTypes: [requestCredentialContent], challenge }
```

Reply:
I didn't want to do this purposely.

I wanted not to include this as people may not know the feature of JS. I spoke with individuals who do not know about the JS syntax. So, keeping it in the assumption they don't know JS.